doc/offline-signing-tutorial.md

```diff
@@ -0,0 +1,259 @@
+{
+  "name": "offline_wallet"
+}
+```
+
+2. Export the offline wallet descriptors to a JSON file named `descriptors.json`. This file will be exported to the online machine to create a watch-only wallet.
+```sh
+[offline]$ ./src/bitcoin-cli -signet -rpcwallet="offline_wallet" listdescriptors | jq -r '.descriptors' >> ~/descriptors.json
+```
+
+### Create the online "watch_only_wallet"
+
+1. On the online machine, create an online watch-only wallet named `watch_only_wallet` without private keys.
+- The `watch_only_wallet` wallet will be used to track transactions received to the `offline_wallet` and for creating PSBT transactions.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="watch_only_wallet" disable_private_keys=true blank=true
+{
+  "name": "watch_only_wallet"
+}
+```
+
+2. Import the `offline_wallet` descriptors to the online `watch_only_wallet` using the `descriptors.json` file created on the offline wallet
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="watch_only_wallet" importdescriptors "$(cat ~/descriptors.json)"
+[
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  }
+]
+```
+
+### Send Funds to the Offline Wallet
+
+1. Generate a new address in the `offline_wallet` to receive funds.
+
+```sh
+[offline]$ ./src/bitcoin-cli -signet -rpcwallet="offline_wallet" getnewaddress
+tb1q6xp92v5jfllewp9gum9mgqlgsfvefrzrvrc8tv
+```
+
+2. Send funds from the `online_sending_wallet` to the generated `offline_wallet` address.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="online_sending_wallet" sendtoaddress "tb1q6xp92v5jfllewp9gum9mgqlgsfvefrzrvrc8tv" 0.0009
+0408d91070a7d02f714066b4b1464f2e0040dce467f8a9edf24255a871a2e865
+```
+
+3. Confirm the received funds (to the `offline_wallet`) using the online `watch_only_wallet`.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="watch_only_wallet" listunspent
+[
+  {
+    "txid": "0408d91070a7d02f714066b4b1464f2e0040dce467f8a9edf24255a871a2e865",
+    "vout": 1,
+    "address": "tb1q6xp92v5jfllewp9gum9mgqlgsfvefrzrvrc8tv",
+    "label": "",
+    "scriptPubKey": "0014d1825532924fff9704a8e6cbb403e88259948c43",
+    "amount": 0.00090000,
+    "confirmations": 1,
+    "spendable": true,
+    "solvable": true,
+    "desc": "wpkh([413037c1/84h/1h/0h/0/0]02b66303a69d660e98c49320afc28dd9abd8f34c9d59c4ca2148a243fa811ea6c8)#6d9cgsep",
+    "parent_descs": [
+      "wpkh([413037c1/84h/1h/0h]tpubDCjzsbgeDYJj2Xgpb5T5CjQ97muocfoQDkYu68RJvu4sspunz82G7pBfFcWUvCGj4AhQoDkbX24jxXyrREyVUAba6B8qKGkFNPyA8GBQ3kA/0/*)#3us38eqx"
+    ],
+    "safe": true
+  }
+]
+```
+
+## Step 2: Sending money from the offline wallet
+
+### Create an Online Receiving Wallet
+
+1. On the online machine, create an online wallet named `online_receiving_wallet`.
```

Comment:
Again I think we can avoid creating yet another wallet here. You could either demonstrate a self-send or get an example destination address from some other source (some faucets have a return address)